libxmp/exempi.py

```diff
@@ -69,7 +69,32 @@ def _load_exempi():
 
     return EXEMPI
 
-EXEMPI = _load_exempi()
+
+class LazyExempi:
+    """This class is the wrapper for the Exempi lib which
```

Comment:
- "This class is" is too much blabla
- "Wrapper for ctypes library making it loaded on actual first use"

Reply:
How can you... I put my sole into this
OK